samples/sample-resource-deployment/package.nls.json

```diff
@@ -52,6 +55,15 @@
 	"wizard.data.controller.agreement": "I accept {0} and {1}.",
 	"contoso.agreement.privacy.statement":"contoso Privacy Statement",
 	"wizard.agreement.contosoCmd.eula":"contoso cmd license terms",
-	"deploy.wizard.action":"Script to notebook"
+	"deploy.wizard.action":"Script to notebook",
+
+	"arc.cores-limit.label": "Cores Limit",
+	"arc.sql.cores-limit.description": "The cores limit of the managed instance as an integer.",
```

Comment:
Let's not use actual product stuff for samples. Just make up some other value like min/max price for something or whatever.

Reply:
kept the name core, but remove references to 'Arc' and 'Sql'